conf/defaults.ini

```diff
@@ -163,12 +163,17 @@ expect_continue_timeout_seconds = 1
 # A value of zero (0) means no limit.
 max_conns_per_host = 0
 
+# Deprecated, use dataproxy_max_idle_connections
 # The maximum number of idle connections that Grafana will keep alive.
 max_idle_connections = 100
 
+# Deprecated, use dataproxy_max_idle_connections
 # The maximum number of idle connections per host that Grafana will keep alive.
 max_idle_connections_per_host = 2
 
+# The maximum number of idle connections per dataproxy that Grafana will keep alive.
+dataproxy_max_idle_connections = 100
```

Comment:
I didn't read the issue this way. I thought we wanted to only use the existing max_idle_connections and remove the per host one. These settings are already under section dataproxy so doesn't make sense to prefix a setting with dataproxy hence why I suggest using the existing setting for both and log a warning if someone uses the per host one. WDYT?

Is 100 still a sensible default for Max idle connections and per host given that per host default was 2 before?

Reply:
By

>I suggest we merge both settings to dataproxy_max_idle_connections and configure both max_idle_connections and max_idle_connections_per_host in the Transport using the new setting. If we add the new setting it could take presence over the old settings and log a deprecation warning when the old settings are used.

I understand that we'll keep both the old config vars and introduce a new one, no?

(cc @bergquist )

Reply:
Though it makes more sense to do it as you say, @marefr

Reply:
Either solution would be fine IMO. I opted for a new setting as it would not change the purpose of the current settings which feels a bit unexpected. But given max_idle_per_host is so new I think @marefr suggestion is better.

Reply:
Lets reuse the max_idle_connections then. If max_idle_connections_per_host is used/configured we log a depreciation warning.